iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -57,7 +57,7 @@ <h1 class="footnote-mode-only" data-i18n="inspector.footnoteProperties">Footnote
 
       <div class="search-controls search-mode-only">
         <div class="search-input">
-          <input id="ixbrl-search" type="text" aria-label="Search iXBRL report"/>
+          <input id="ixbrl-search" type="text" aria-label="Search iXBRL report" disabled="disabled" />
```

Comment:
`disabled="disabled"` should just be `disabled` I think since it's a boolean attribute.

Reply:
Yes, that's correct.  I know that we have had issues with viewer files not being XHTML-compatible in the past, so I've been aiming for XHTML-compatible throughout.  Interestingly, browsers don't seem to care about this issue here (`disabled` vs `disabled="disabled"`) but they do complain about using `&nbsp;` if you open the viewer in XHTML mode.

Reply:
That makes sense, thanks for the explanation!